xarray/core/missing.py

```diff
@@ -743,6 +740,8 @@ def interp_func(var, x, new_x, method, kwargs):
             concatenate=True,
             dtype=dtype,
             new_axes=new_axes,
+            # meta=var._meta,  # TODO: uncomment when dask is bumped
```

Comment:
```suggestion
            # TODO: uncomment when min dask version is > 2.15:
            # meta=var._meta,  
```
Specifying a version helps other readers to take action, otherwise they'll always wait for the next bump. :)